### ISSUE TITLE
Saved token is always invalid?

### ISSUE THREAD

**Issue by lublak** [kanimaru/twitcher#29]
![grafik](https://github.com/user-attachments/assets/c305fe1f-ff7d-4ffd-9303-6a740b2e320e)

**Comment by kanimaru** [comment]
What kind of Auth Flow you are using?

**Comment by lublak** [comment]
@kanimaru 
![grafik](https://github.com/user-attachments/assets/1e4c2972-aa08-4619-a1a8-a27cdd96d208)

**Comment by kanimaru** [comment]
I think I fixed it with the latest changes. Can you check it?

**Comment by lublak** [comment]
> I think I fixed it with the latest changes. Can you check it?

I will try it hopefully this weekend!

**Comment by lublak** [comment]
@kanimaru 
Yes, I'm sorry, I'm a bit stressed at the moment and unfortunately haven't been able to think about the project.
But I think if you have tested it yourself it should fit. I'll have to see if I can find the time again. Sorry for the inconvenience.
